README.md

```diff
@@ -9,16 +9,19 @@ This project intends to provide 2 packages to enable JNI interop from Dart & Flu
 | [jni](jni/) | Ergonomic C bindings to JNI C API and several helper methods |
 | [jnigen](jnigen/) | Tool to generate Dart bindings to Java code using FFI |
 
-This is a work-in-progress project under Google Summer of Code 2022 program.
+This project is a work-in-progress and highly experimental. Users can check out and experiment with the examples. We do not guarantee stability of the API or commands at this point.
 
 ## SDK Requirements
 Dart standalone target is supported, but due to some problems with pubspec, the `dart` command must be from Flutter SDK and not Dart SDK. See [dart-lang/pub#3563](https://github.com/dart-lang/pub/issues/3563).
 
+Along with JDK, maven (`mvn` command) is required. On Windows, it can be installed using a package manager such as `scoop`.
```

Comment:
In the Dart ecosystem we recommend Chocolatey (see https://dart.dev/get-dart#install -> Windows). So maybe refer to that instead.

Reply:
I will mention both, because scoop is much cleaner these days.